src/EnergyPlus/WaterToAirHeatPumpSimple.cc

```diff
@@ -2367,7 +2378,14 @@ namespace WaterToAirHeatPumpSimple {
         QSensible *= PartLoadRatio;
         Winput *= RuntimeFrac;
         QSource = QSource_fullload * PartLoadRatio;
-
+        DataHeatBalance::HeatReclaimSimple_WAHPCoil(HPNum).AvailCapacity = QSource;
+        for (DesuperheaterNum = 1; DesuperheaterNum <= NumWaterHeaterDesuperheater; ++DesuperheaterNum) {
+            if (UtilityRoutines::SameString(WaterHeaterDesuperheater(DesuperheaterNum).HeatingSourceType,"COIL:COOLING:WATERTOAIRHEATPUMP:EQUATIONFIT") 
+                && UtilityRoutines::SameString(WaterHeaterDesuperheater(DesuperheaterNum).HeatingSourceName,SimpleWatertoAirHP(HPNum).Name)) {
+                WHDSHT_Reclaimed_Heat += WaterHeaterDesuperheater(DesuperheaterNum).HeaterRate;
+            }
+        }
+        
```

Comment:
Yes, this looks good and should show a small impact on the plant. I just wish I could figure out how to do something like this for the improved air-side heat transfer.

Reply:
Hi, Richard, thanks, I found I didn't give initial value for each iteration so I fixed it. Now it does run successfully and give some impact on plant.